ao_mail/models/res_partner.py

```diff
@@ -1,40 +1,24 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2016-2017 Akretion (http://www.akretion.com)
 # @author: Alexis de Lattre <alexis.delattre@akretion.com>
 # Copyright 2017-18 Eficent Business and IT Consulting Services S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models
+from odoo import api, models
 
 
-class ResPartner(models.Model):
+class Partner(models.Model):
     _inherit = 'res.partner'
 
-    notify_email = fields.Selection(
```

Comment:
this is migrated to field notification_type in user, no need to create a boolean in user.